src/simple_commander/main.py

```diff
@@ -466,6 +505,7 @@ def stop_rotate(self, hero):
         hero.rotate_is_pressing = False
 
     def remove_from_ignore(self, hero_id):
```

Comment:
I don't think you need any ignore or exclusion lists for units. It's very hard to read the logic of code with such properties